examples/ng9/src/app/assets-image/assets-image.component.html

```diff
@@ -0,0 +1 @@
+<img src="/assets/cypress.png" />
```

Comment:
`/src/assets/cypress.png` actually loads the image

Reply:
You're right but path should really need to begin with `assets`

Reply:
Okay cool. I noticed that assets aren't loaded the same way as doing an ng build. A lot of them missing so the component doesn't render the same way as localhost.

Reply:
@LeJeanbono any progress on this issue?

Reply:
Tried something with https://github.com/angular/ngtools-webpack-builds but no success...

Reply:
:(

Reply:
It work with `/assets/cypress.png` but the src should be `assets/cypress.png` (not working)

Reply:
Perhaps @bahmutov could help?
Would this solution work for all assets under the assets folder?
Also is it possible to just load all styles when mounting a component without having to `setConfig`? Cos we have quite a few styles under the assets folder.

Reply:
@LeJeanbono you manage?

Reply:
@LeJeanbono nothing?

Reply:
This is what I am seeing when running the test
![Screenshot 2020-10-28 09:28:42](https://user-images.githubusercontent.com/13921692/97405230-02a51600-1900-11eb-950b-187749b703ea.png)